.github/workflows/build.yml

```diff
@@ -206,6 +206,11 @@ jobs:
     name: Integration tests
     needs: build
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # try the latest stable and potential pre-releases
+        jupyterlite_version: ["jupyterlite -U", "jupyterlite --pre"]
```

Comment:
This should make it easier to catch potential breaking changes that may be landing upstream in JupyterLite.

Reply:
If such breaking changes happen in JupyterLite in the future and break the CI here in this repo, this could be commented and tracked in a separate issue.